### ISSUE TITLE
SFTP repo init failed

### ISSUE THREAD

**Issue by 917Oliver** [kmwoley/restic-windows-backup#50]
Local-Backups are running fine with this Script .Now I tried using this script for running backups over SFTP. 
I changed the repo (in secrets.ps1) to **$Env:RESTIC_REPOSITORY='sftp:sftp@192.168.50.95:/upload'** (yes, the user is called sftp).

Using Restic with SFTP manually runs without a problem. But when using the install-script, I get the following error: 
**Fatal: create repository at sftp:sftp@192.168.50.95: failed: cmd.Start: cmd.Start: exec: "ssh": executable file not found in %PATH%**
I added the OpenSSH .exe to Path (%SYSTEMROOT%\System32\OpenSSH\ssh.exe) and also tried it throug the terminal.

Did I miss something or do I have to configure something else?
The SFTP-Connection is with a public-key, so that no password for the connection is needed. This works fine if I back up usig restic manually.

**Comment by dicer** [comment]
What Windows version are you using? My Win10 seems to have ssh preinstalled and I didn't install anything extra.
If you installed it yourself, make sure that it is in _standard_ path not a path you configured! This is because the backup task is run with the SYSTEM account and not your user. That's probably why it complains about ssh not being in the path (cause it isn't for SYSTEM).
I documented how I got sftp running (there are more pitfalls you haven't run into yet ;)): https://github.com/kmwoley/restic-windows-backup/pull/52
